android/java/res/layout/fragment_creating_wallet.xml

```diff
@@ -24,25 +23,24 @@
             android:layout_gravity="center_horizontal" />
 
         <TextView
-            style="@style/BraveWalletTextViewTitle"
+            style="@style/HeadingH3"
             android:layout_width="match_parent"
+            android:layout_height="wrap_content"
             android:gravity="center_horizontal"
             android:layout_marginHorizontal="32dp"
             android:layout_marginBottom="8dp"
             android:text="@string/creating_wallet_onboarding_text"
             android:textColor="@color/wallet_text_color"
```

Comment:
Sorry missed it in initial review but i think we should change it.

Reply:
Done in commit 👉 [`a85fb75` (#24063)](https://github.com/brave/brave-core/pull/24063/commits/a85fb75ba0e9f37da89ed68e35e9141a90eadbdf)